tests/flow/test_rdbs.py

```diff
@@ -69,6 +70,15 @@ def testRDBCompatibility():
         newDbFileName = r.execute_command('config', 'get', 'dbfilename')[1].decode('ascii')
         env.assertEqual(newDbFileName, dbFileName)
         OLD_KEYS.sort()
+
+        if(fileName == "1.4.9_with_avg_ctx.rdb"):
```

Comment:
@OfirMos how did you create the file?

Reply:
@danni-m This is actually strange since there aren't suppose to be any rules after the restore completes